easybuild/easyconfigs/r/rstudio/rstudio-1.2.5033-fosscuda-2019b-Java-11.eb

```diff
@@ -38,7 +38,9 @@ dependencies = [
     ('Java', '11', '', True),
 ]
 
-osdependencies = ['pam-devel']
+osdependencies = [
+    ('pam-devel', 'libpam0g-dev')
```

Comment:
@crubb Is it worth adding an additional constant for this in framework, like we did for OpenSSL packages in https://github.com/easybuilders/easybuild-framework/pull/3309 ?